api/pkg/db/model/model.go

```diff
@@ -80,6 +80,7 @@ type (
 		Description         string
 		URL                 string `gorm:"not null;default:null"`
 		DisplayName         string
+		Deprecated          bool   `gorm:"default:false"`
```

Comment:
In the table we are setting it to false right ? So above instead of `null` should we return `false`